crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -404,23 +417,34 @@ impl RoomEventCacheInner {
             self.pagination_batch_token_notifier.notify_one();
         }
 
+        let mut events_update_the_timeline = false;
```

Comment:
Always feels weird to me to have a boolean that's a sentence; usually I'd advocate for `verb_nouns`. How about `trigger_timeline_update`, or `forward_generic_update`?

Reply:
I've renamed to `update_has_been_sent`.